lib/wire-protocol.d.ts

```diff
@@ -50,6 +50,7 @@ export interface QueryRequestHeaders {
      */
     traceparent?: string;
 }
+/** tagged declares that type information is transmitted and received by the driver. "simple" indicates it is not. */
```

Comment:
i'm confused about why this just came in now.

Reply:
oh!  I bet becayse the last commit of the previous PR was done using the `commit suggestion` button

Reply:
oh yeah good call.

Reply:
we need to clean up the build process to get it out of the commit hook 😵‍💫